src/user/models/kyc/kyc-webhook.service.ts

```diff
@@ -74,15 +75,24 @@ export class KycWebhookService {
             city: userData.location,
             zip: userData.zip,
             phone: userData.phone,
-            //TODO change for KYC Update v2
-            kycStatus: KycCompleted(userData.kycStatus) ? KycWebhookStatus.FULL : KycWebhookStatus.NA,
+            kycStatus:
+              KycCompleted(userData.kycStatus) && spiderData?.chatbotResult
+                ? KycWebhookStatus.FULL
+                : KycCompleted(userData.kycStatus)
+                ? KycWebhookStatus.LIGHT
+                : userData.kycStatus === KycStatus.REJECTED
+                ? KycWebhookStatus.REJECTED
+                : KycWebhookStatus.NA,
             kycHash: userData.kycHash,
           },
           reason: reason,
         };
 
-        await this.http.post(`${walletUser.apiUrl}/kyc/update`, data, {
-          headers: { 'x-api-key': Config.lock.apiKey },
+        const apiKey = this.walletService.getApiKeyInternal(user.wallet.name);
+        if (!apiKey) throw new ConflictException(`ApiKey for wallet ${user.wallet.name} not available`);
```

Comment:
I would throw a standard error (`throw new Error(...)`), because this method is normally called from within our API. So we don't need HTTP codes.